src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -301,20 +513,21 @@ export default class ArrivalBase {
 
     /**
      * Determine delay until next spawn
+     *
+     * @for ArrivalBase
+     * @method nextInterval
+     * @return {number}
      */
     nextInterval() {
-        // TODO: these next 3 vars could easily be moved to a constants file
-        // nautical miles
-        const min_entrail = 5.5;
-        // in seconds
-        const min_interval = min_entrail * (3600 / this.speed);
-        let tgt_interval = 3600 / this.frequency;
```

Comment:
yup, that would work. almost all time units I've stumbled across are stored here in seconds, for example `prop.game.time` (or whatever it may be called now) and the various timeouts are all second-based.